### ISSUE TITLE
Referências bibliográficas na plataforma

### ISSUE THREAD

**Issue by hjbortol** [livro-aberto/BookCloud#183]
Caros, não dá para deixar mais bonitinha a referência a uma referência bibliográfica na plataforma? Digo, se espaços não são permitidos, fica muito feio o ano colado no nome do autor. E quando o artigo tiver mais de um autor? Vamos usar vírgulas sem espaços, tudo junto e colado?

![image](https://user-images.githubusercontent.com/7694624/29171301-7529f7fa-7db2-11e7-8da3-b7bf27a83467.png)

Penso que essas questões têm que ser resolvidas adequadamente para evitar o que aconteceu no piloto de frações: vocês terem que olhar e modificar diretamente no código LaTeX gerado.

**Comment by hjbortol** [comment]
Nota: o exemplo dado na plataforma (syntax) não está seguindo o estilo apalike que escolhemos. Deveria ser assim:

![image](https://user-images.githubusercontent.com/7694624/29171454-0ddac5a6-7db3-11e7-9c98-eb65c31c00bf.png)

**Comment by hjbortol** [comment]
Caros (incluo os demais na conversa, pois é uma decisão conjunta): @cambrainha, @wmrezende, @DirceUesu, @GladsonAntunes, @mpfaraujo [Cydara e Letícia ainda não estão registradas]

Na lista de referências do estilo APA para referências (o que está sendo proposto para usarmos), há uma caracerística que não gosto: apenas o sobrenome do autor é escrito, mas os nomes são abreviados. Acho melhor colocar indicar os nomes, pois isso pode facilitar a busca pela referência.

O que acham?

Minha sugestão: ou (1) Manter o estilo APA, mas não abreviando os nomes ou (2) usar outro estilo (pode ser a ABNT, mas sem citações com nomes em caixa alta).

**Comment by simas0** [comment]
ABNT é como está abaixo. Eu estou de acordo, mas não sei como implementar no Sphix. Precisamos de orientação @gutosurrex .

> Segundo Vaz (2008, p. 63) “não saber usar a internet em um futuro próximo será como não saber abrir um livro ou acender um fogão, não sabermos algo que nos permita viver a cidadania na sua completitude”.

**Comment by gutosurrex** [comment]
Como o Sphinx não permite espaços em rótulos, eu recomendo `[Einstein-1905]` ou `[Einstein_1905]`. O que vocês acham? Tem também uma biblioteca para bibtex no sphinx, podemos considerá-la.

**Comment by hjbortol** [comment]
E quando a referência tiver mais de um autor? Este esquema vai ficar feio:

[Einstein,Turing,1945] (sem espaços; Sphynx aceita rótulos com vírgulas?)

**Comment by simas0** [comment]
Sim, aceita vírgulas, mas acho que fica melhor [Einstein-et-al-1945]. Melhor ainda é sem os colchetes.

_Enviado do meu Lenovo K33b36 usando [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by hjbortol** [comment]
Não tem um esquema de usar "\ " (sem as aspas) para indicar um espaço em branco? Isso não funciona em labels?

**Comment by simas0** [comment]
Sphinx não permite retirar os colchetes para representar uma citação. Estou usando hífen. Consertei a sintaxe e dei instruções para se obter o formato APA no Google Acadêmico.

**Comment by hjbortol** [comment]
No arquivo PDF também vai ficar com hífen?

**Comment by simas0** [comment]
Sim, veja os distratores em vetores no ramo_simas. A referência foi para o final do PDF.

_Enviado do meu Lenovo K33b36 usando [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by hjbortol** [comment]
Hummm... Fica estranho. 

:+1: para o LaTeX, :-1: para o Sphynx ... 😃

**Comment by gutosurrex** [comment]
Eu diria que é fácil mudar no pdf, gastaria um tempo para mudar no HTML.
